src/pvnet_app/data/nwp.py

```diff
@@ -222,5 +222,5 @@ def rename_ecmwf_variables():
         d = d.compute()
 
         # save back to path
-        os.system(f"rm -rf {nwp_ecmwf_path}")
+        shutil.rmtree(nwp_ecmwf_path, ignore_errors=True)
```

Comment:
thanks for this

Reply:
Shouldn't be doing this system call stuff anywhere when avoidable!